src/GraphQL/Validation/Rules/ArgumentsOfCorrectType.cs

```diff
@@ -18,11 +18,7 @@ public class ArgumentsOfCorrectType : IValidationRule
         /// </summary>
         public static readonly ArgumentsOfCorrectType Instance = new ArgumentsOfCorrectType();
 
-        /// <inheritdoc/>
-        /// <exception cref="ArgumentsOfCorrectTypeError"/>
```

Comment:
Generally speaking, this does not correspond to reality, because all such methods do not throw mentioned exceptions.

Reply:
I know, but I thought it was useful.

Reply:
agree